Makefile

```
@@ -24,10 +24,10 @@ clean:
 	rm -f *deb *rpm
 	rm -f sha256sum md5sum
 
-$(NAME):
+$(NAME): $(wildcard **/*.go *.go)
```

It doesn't go recursively, unfortunately. Only to the closest subdirectories.

```suggestion
$(NAME): $(shell find . -name '*.go')
```

This shell-function does the magic we need